MAINTAINERS.md

```diff
@@ -4,7 +4,7 @@ This document contains instructions and guidelines for maintainer tasks, like pu
 
 ## Continuous deployment
 
-Whenever a PR is merged into `master`, Travis will automatically run the scripts that build the app and upload the results to Netlify. Netlify will then deploy the site. We currently allow only one Travis job at a time so that concurrent deploy commands do not cause a conflict.
```

Comment:
In [.travis.yml](https://github.com/ember-learn/guides-source/blob/85c33ca09f41b49a6d3abda9eea1735a610e9e56/.travis.yml#L18), I couldn't find evidence that Travis does something to build the app and upload the results to Netlify. It does have `JOBS=1`. Maybe `secure`?

To my understanding, Netlify watches for a successful merge before deploying the app.